app/src/main/java/com/sopt/now/ui/login/LoginActivity.kt

```diff
@@ -71,29 +47,59 @@ class LoginActivity : BindingActivity<ActivityLoginBinding>(R.layout.activity_lo
         }
     }
 
-    private fun validateLogin(): Boolean {
-        return validateId() && validatePassword()
+    private fun initButton() {
+        binding.btnLoginToSignup.setOnClickListener {
+            initSignUpBtnClickListener()
+        }
+
+        binding.btnLogin.setOnClickListener {
+            loginButtonEvent()
+        }
+    }
+
+    private fun initObserver() {
+        viewModel.liveData.observe(this) { state ->
+            when (state) {
+                is UiState.Loading -> {
+
+                }
+
+                is UiState.Success -> {
+                    navToHome(state.data)
+                }
+
+                is UiState.Error -> {
+                    toast(state.errorMessage)
+                }
```

Comment:
```suggestion
                is UiState.Success -> navToHome(state.data)
                is UiState.Error -> toast(state.errorMessage)
```
사실 상관없는데 이게 가독성이 더 좋아보여요! 상관은 없습니다

Reply:
엇 맞네요 한줄인데! 변경했습니다 감사해요!